src/XrdPlugins.cmake

```diff
@@ -133,6 +134,22 @@ target_link_libraries(
   XrdServer
   XrdUtils )
 
+#-------------------------------------------------------------------------------
+# An OSS plugin for calculating storage performance statistics
+#-------------------------------------------------------------------------------
+add_library(
+  ${LIB_XRD_OSSSTATS}
+  MODULE
+  XrdOssStats/XrdStatsConfig.cc     XrdOssStats/XrdStatsConfig.hh
```

Comment:
The names here do not follow the standard naming conventopn. The directory name is the prefix of whatever files are in the directory (well, hh and cc files). So, if youi want XrdStatsConfig,cc in XrdOssStats then it really should be named XrdOssStatsConfig.cc. So please introducs the"Oss" into the file names to be consistent with that policy.

And, yes, there are some inconsistencies in the repo that are historical in nature and not worth changing but we don't want to add to it.

Reply:
Fixed in latest commit!